README.md

```diff
@@ -1289,6 +1289,90 @@ mysql> select * from memory_global_total;
 +-----------------+
 ```
 
+#### metrics / metrics_56
+
+##### Description
+
+Creates a union of the following information:
+
+   *  performance_schema.global_status (information_schema.GLOBAL_STATUS for metrics_56)
+   *  information_schema.INNODB_METRICS
+   *  Performance Schema global memory usage information
+   *  Current time
+
+The difference betweem the metrics and the metrics_56 views are whether the global status is taken from performance_schema.global_status instead of
+from the Information Schema.Use the metrics view if the MySQL version is 5.7.6 or later and show_compatibility_56 = OFF. Otherwise use metrics_56.
+See also https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_show_compatibility_56
+
+In MySQL 5.7.6 and later the metrics_56 view will generate one warning that INFORMATION_SCHEMA.GLOBAL_STATUS is deprectatedd.
+
+For view has the following columns:
+
+   * Variable_name: The name of the variable
+   * Variable_value: The value of the variable
+   * Type: The type of the variable. This will depend on the source, e.g. Global Status, InnoDB Metrics - ..., etc.
+   * Enabled: Whether the variable is enabled or not. Possible values are 'YES', 'NO', 'PARTIAL'.
```

Comment:
It would be nice to explain in what case PARTIAL can be returned (looks like "a subset of memory instruments are enabled in Performance schema").